Issue 1:

   when ploting pgn I did not enter a valid pgn i just hit an arrow key and then enter and causing the program to crash
Issue 2:
   Installing issues:
	needed to run to get tinker
		sudo apt-get install ptython3-tk
  	needed to get protobuf
		pip3 install protobuf	
	needed to get net-tools
		sudo apt-get install net-tools

Issue 3:
   compare logs returns:
   Traceback (most recent call last):
  File "/usr/local/bin/polycan", line 11, in <module>
    load_entry_point('polycan', 'console_scripts', 'polycan')()
  File "/home/sam/polycan/polycan/main.py", line 111, in main
    main_menu()
  File "/home/sam/polycan/polycan/main.py", line 98, in main_menu
    compare_logs(uploaded_logs, known, helper)
  File "/home/sam/polycan/polycan/log_handler.py", line 511, in compare_logs
    log1_Name = find_log()
  File "/home/sam/polycan/polycan/log_handler.py", line 165, in find_log
    names = get_lognames()
  File "/home/sam/polycan/polycan/sql_interface.py", line 114, in get_lognames
    return database.list_logs()
   NameError: name 'database' is not defined 

Issue 4:
   when sending or capturing logs. it depends on what directory you are in when you runt he poly command as the path structure is realtive
